### DISCUSSION TITLE
Transmission Permission Denied

### DISCUSSION THREAD

**Discussion by FarleighRed** [tteck/Proxmox#2954] [Help Needed]
I have successfully added Transmission as a container and mount my NAS as a CIFS share successfully to the container. I am am to view files and write files to the NAS using a NANO command so everything seems to be working fine.

I cant however get a transmission download to save on the NAS whenever I try to do this I get a Permission denied message in the transmission UI and it fails.

Any help would be appreciated in resolving this one.

**Comment by arisermpo87** [comment]
Same exact problem here. Did you figure it out?

**Comment by regystro** [comment]
I was getting permission denied until I mounted the SMB/CIFS share this way: https://dev.to/oswllt/proxmox-bind-smb-mount-to-unprivileged-lxc-container-the-easy-way-3l4k
I used the first way "The Secured way via /etc/fstab file" for an unprivileged LXC container.
